### DISCUSSION TITLE
TShock 4.5.17

### DISCUSSION THREAD

**Discussion by hakusaro** [Pryaxis/TShock#2602] [Show and tell]
This release fixes two really annoying bugs that were contributed by @gohjoseph!

## Notable changes
* Fixed duplicate characters (twins) after repeatedly logging in as the same character due to connection not being immediately closed during `NetHooks_NameCollision`. (@gohjoseph)
* Fixed mobs not dropping picked up coins. (@gohjoseph)

## Upgrading
To upgrade TShock, replace all of the files in the location of your server installation with all of the files from the release zip we provide. Overwrite all files. No data will be lost.

<hr /><em>This discussion was created from the release <a href='https://github.com/Pryaxis/TShock/releases/tag/v4.5.17'>TShock 4.5.17</a>.</em>

**Comment by su-ziye** [comment]
Why does the Tshock server become a world birth point when I set the birth point with the bed and re-enter the world?

**Comment by Shunsuke-Yoshino** [comment]
[Normal Boss Bags](https://github.com/Quinci135/NormalBossBags) by Quinci don't working at this version :c 
But.. working at 4.5.13 (1.4.3.2)
